{{cookiecutter.project_slug}}/_/development/java/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM openjdk:8-jdk-alpine as build
+
+WORKDIR /workspace/app
+
+COPY . .
```

Comment:
This should then be a threat only on local development environments. The automated builds run based on what is in version control, hence transparently visible. Do you see a threat here, too?

Reply:
It's just a best practice in general. I will let you decide if a change is necessary.

Reply:
If it's best practice it's best best practice for a reason. If that reason applies to our case then we should follow the recommendation. If it doesn't (e.g. because the setting is different, hence the bespoken threat does not apply) then it might be safe to keep the approach. Which case do you see applicable here?

Reply:
I would avoid even in this case the . . approach even though it seems transparent. There's always the possibility that we copy too much into the image.